src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -35,12 +36,14 @@ public string UniqueId
             }
         }
 
-        private WorkspaceDependencyView DependencyView
+        internal WorkspaceDependencyView DependencyView
         {
             get;
             set;
         }
 
+        internal ViewStartupParams ViewLoadedParams;
+
         private ReadyParams ReadyParams;
```

Comment:
in general I think cacheing these is kind of worrisome, I have no idea but seems like it might cause some kind of reference path issue (memory leak)